src/main/steps/applicant1/confirm-your-joint-application/content.ts

```diff
@@ -1,158 +1,176 @@
+import config from 'config';
+
 import { getFormattedDate } from '../../../app/case/answers/formatDate';
-import { getAnswerRows } from '../../../app/case/answers/getAnswerRows';
-import { Checkbox } from '../../../app/case/case';
-import { ChangedNameHow, YesOrNo } from '../../../app/case/definition';
+import { CaseWithId, Checkbox } from '../../../app/case/case';
+import { YesOrNo } from '../../../app/case/definition';
 import { TranslationFn } from '../../../app/controller/GetController';
 import { FormContent } from '../../../app/form/Form';
 import { isFieldFilledIn } from '../../../app/form/validation';
-import { connectionBulletPointsTextForSoleAndJoint } from '../../../app/jurisdiction/bulletedPointsContent';
+import { connectionBulletPointsTextForRespondent } from '../../../app/jurisdiction/bulletedPointsContent';
 import { jurisdictionMoreDetailsContent } from '../../../steps/applicant1/connection-summary/content';
-import { Sections } from '../../applicant1Sequence';
-import { moreDetailsComponent } from '../../applicant2/check-your-joint-application/content';
 import { CommonContent } from '../../common/common.content';
-import * as urls from '../../urls';
 
-const en = ({ isDivorce, partner, userCase, marriage, civilPartnership }: CommonContent) => ({
+const isSubmit = (isApplicant2: boolean, userCase: Partial<CaseWithId> | undefined): boolean => {
+  return (
+    isApplicant2 ||
+    (!isApplicant2 &&
```

Comment:
since you checked if it's **isApplicant2** above then I don't think you don't need to add in the **!isApplicant2** bit as well

Reply:
Again preference.  Yes you are right but I do find it easier to read that I'm not dealing with Applicant 2 by putting that there (so I'm dealing with Applicant 1).  Easier to read in my head.  I have removed though